data/definitions.json

```diff
@@ -945,23 +945,29 @@
 			"alias": "sesp-exif-languagecode",
 			"label": "Exif:Language code"
 		},
-                "GPSLATITUDE": {
-                        "id": "___EXIFGPSLATITUDE",
-                        "type": "_txt",
-                        "alias": "sesp-exif-gpslatitude",
-                        "label": "Exif:GPSLatitude"
-                },
-                "GPSLONGITUDE": {
-                        "id": "___EXIFGPSLONGITUDE",
-                        "type": "_txt",
-                        "alias": "sesp-exif-gpslongitude",
-                        "label": "Exif:GPSLongitude"
-                },
-                "GPSALTITUDE": {
-                        "id": "___EXIFGPSALTITUDE",
-                        "type": "_txt",
-                        "alias": "sesp-exif-gpsaltitude",
-                        "label": "Exif:GPSAltitude"
-                }
+		"GPSLATITUDE": {
+				"id": "___EXIFGPSLATITUDE",
+				"type": "_txt",
+				"alias": "sesp-exif-gpslatitude",
+				"label": "Exif:GPSLatitude"
+		},
+		"GPSLONGITUDE": {
+				"id": "___EXIFGPSLONGITUDE",
+				"type": "_txt",
+				"alias": "sesp-exif-gpslongitude",
+				"label": "Exif:GPSLongitude"
+		},
+		"GPSALTITUDE": {
+				"id": "___EXIFGPSALTITUDE",
+				"type": "_txt",
+				"alias": "sesp-exif-gpsaltitude",
+				"label": "Exif:GPSAltitude"
+		},
+		"_LINKSTO": {
```

Comment:
Why the prefix?

Reply:
That was a copy-paste artifact form my local definition